ihatemoney/tests/tests.py

```diff
@@ -959,6 +973,8 @@ def test_statistics(self):
             response.data.decode("utf-8"),
         )
 
+        # TODO: Add a refund !
```

Comment:
Why's this todo here?

Reply:
Wow seems like I didn't look at my PR correctly before opening this PR :disappointed: